src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java

```diff
@@ -0,0 +1,239 @@
+        gridBagConstraints.ipadx = 50;
+        gridBagConstraints.ipady = 0;
+        panelTeamServer.add(buttonGenerate, gridBagConstraints);
+
+        buttonGroupUpgradeType = new ButtonGroup();
+        jRadioButtonFullUpgrade = new JRadioButton(Constants.FULL_UPGRADE);
+        jRadioButtonPatchUpgrade = new JRadioButton(Constants.PATCH_UPGRADE);
+        buttonGroupUpgradeType.add(jRadioButtonFullUpgrade);
+        buttonGroupUpgradeType.add(jRadioButtonPatchUpgrade);
+
+        gridBagConstraints.gridwidth = 1;
+        gridBagConstraints.weightx = .5;
+        gridBagConstraints.weighty = .2;
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.ipadx = 0;
+        gridBagConstraints.gridy = 5;
+        panelTeamServer.add(jRadioButtonFullUpgrade, gridBagConstraints);
+
+        gridBagConstraints.gridwidth = 1;
+        gridBagConstraints.weightx = .5;
+        gridBagConstraints.weighty = .2;
+        gridBagConstraints.gridx = 1;
+        gridBagConstraints.gridy = 5;
+        panelTeamServer.add(jRadioButtonPatchUpgrade, gridBagConstraints);
+
+        buttonChangeVersion = new JButton(Constants.CHANGE_VERSION_NUMBER);
+        buttonReset = new JButton(Constants.RESET);
+
+        gridBagConstraints.gridwidth = 1;
+        gridBagConstraints.weightx = .5;
+        gridBagConstraints.weighty = .2;
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 6;
+        gridBagConstraints.ipadx = 0;
+        panelTeamServer.add(buttonChangeVersion, gridBagConstraints);
+
+        gridBagConstraints.gridwidth = 1;
+        gridBagConstraints.weightx = .5;
+        gridBagConstraints.weighty = .2;
+        gridBagConstraints.gridx = 1;
+        gridBagConstraints.gridy = 6;
+        gridBagConstraints.ipadx = 100;
+        panelTeamServer.add(buttonReset, gridBagConstraints);
+
+        textAreaStatus = new JTextArea();
+        gridBagConstraints.fill = GridBagConstraints.HORIZONTAL;
+        gridBagConstraints.ipady = 100;
+        gridBagConstraints.weightx = 0.0;
+        gridBagConstraints.gridwidth = 3;
+        gridBagConstraints.gridx = 0;
+        gridBagConstraints.gridy = 7;
+        panelTeamServer.add(textAreaStatus, gridBagConstraints);
+        textAreaStatus.setBorder(BorderFactory.createLineBorder(Color.BLACK));
+
+        return panelTeamServer;
+    }
+
+    public JPanel initDataArchitectUI() {
+        panelDataArchitect = new JPanel();
+        return panelDataArchitect;
+    }
+
+    public JPanel initHelpUI() {
+        panelHelp = new JPanel();
+        return panelHelp;
+    }
+
+    //on click the of browse button
+    public void onPressedBrowse() {
+        JFileChooser j = new JFileChooser(FileSystemView.getFileSystemView().getHomeDirectory());
+
+        // set the selection mode to directories only
+        j.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);
+
+        // invoke the showsOpenDialog function to show the save dialog
+        int r = j.showOpenDialog(null);
```

Comment:
bad naming of variable. Based on what showOpenDialog returns. rename r.

Reply:
done